File: Server Side/StudyStipendCalcAPI/StudyStipendCalcAPI/DTOs/CalculateDto.cs
using System;
using System.Collections.Generic;
using StudyStipendCalcAPI.DTOs.StudentDtos;

namespace StudyStipendCalcAPI.DTOs
{
    public class CalculateDto
    {
        public List<StudentDto> students;
        public int input;
        public double minStipendIndex;
        public int minPrice;
        public int maxPrice;
    }
}

File: Client side/App/src/app/_models/Other/CalculateDto.ts
import { StudentCalculationDto } from '../Student/StudentCalculationDto';

//Ösztöndíjszámoláshoz szükséges adatokat tartalmazó object
export class CalculateDto {
  constructor(
    public maxPrice: number,
    public minPrice: number,
    public minStipendIndex: number,
    public input: number,
    public students: StudentCalculationDto[],
  ) {}
}
